Sources/ConsoleKit/Command/Async/AsyncCommand.swift

```diff
@@ -79,33 +79,21 @@
 ///                        ||     ||
 ///
 #if swift(>=5.5) && canImport(_Concurrency)
+import NIO
+
 @available(macOS 12, iOS 15, watchOS 8, tvOS 15, *)
-public protocol AsyncCommand: AnyAsyncCommand {
-    associatedtype Signature: CommandSignature
-    func run(using context: CommandContext, signature: Signature) async throws
+public protocol AsyncCommand: Command {
+    func run(usingAsync context: CommandContext, signature: Signature) async throws
```

Comment:
This can be `func run(using context: CommandContext, signature: Signature) async throws`.